Add docs for prometheus alert manager to corollate to grafana (#87)
komodorio/docs: docs/Integrations/AlertManager.md

# Prometheus/Grafana Alert Manager Integration

> Integration with alert manager reports alerts also to Komodor. We connect the alert to the Kubernetes workload and display the events on the timeline.

Reviewer: ```suggestion
The Integration of Grafana / Prometheus AlertManager allows alerts triggered by AlertManager to appear on Komodor timelines. 
```

---

## Installation

> ### The Alert Manager integration involves 3 parts:

Reviewer: ```suggestion
### The Alert Manager integration involves 3 steps:
```

1. Enabling the integration in Komodor.
2. Creating webhook on the alert manager / Grafana.
3. Adding labels to the alert.

---

### 1. Enabling the integration in Komodor

> To enable the Komodor Prometheus Alert Manager integration go to [Komodor integrations page](https://app.komodor.com/main/integration) and select Prometheus/Grafana Alert Manager.

Reviewer: ```suggestion
To enable the Komodor Prometheus Alert Manager integration go to [Komodor integrations page](https://app.komodor.com/main/integration), select Prometheus/Grafana Alert Manager, and install the integration. 
```

        send_resolved: true
```

### 2. (Option B) Configure alerts from Grafana

1. Go to `Alerting` -> `Contact points`.

   ![AlertingContactPoints](./img/AlertManagerGrafana/alerting_contact_points.png)

2. Click `New contact point`.

   ![NewContactPoint](./img/AlertManagerGrafana/new_contact_point.png)

3. Enter in name `komodor`, in `Contact point type` select `Webhook` and in `Url` insert the URL from the UI.

   ![ConfigureNewContactPoint](./img/AlertManagerGrafana/configure_new_contact_point.png)

4. Click `Alerting` -> `Notification policies`.

![NotificationPolicies](./img/AlertManagerGrafana/notification_policies.png)

5. In the notification policies, configure komodor contact endpoint as you wish to configure alerts to Komodor.

### 3. Adding labels to the alert.

> 1. Please note to specify 2 **labels** on the alert in order to connect them to the Kubernetes workload:

Reviewer: ```suggestion

To relate the alert to the relevant workload and make the alerts visible on workloads timelines -  adding labels to alerts is required. Each alert without a label will be added to the system without mapping to a specific service.

1. Please note to specify 2 **labels** on the alert in order to connect them to the Kubernetes workload:
```
